services/utils/config.py

```
@@ -0,0 +1,43 @@
+from pydantic_settings import BaseSettings, SettingsConfigDict
+from typing import Optional
+
+class Settings(BaseSettings):
+    REDIS_HOST: str = "localhost"
+    REDIS_PORT: int = 6379
+    REDIS_TIMEOUT_S: float = 2.0
+    API_HOST: str = "0.0.0.0"
```

Ruff (0.16.0)

[error] Possible binding to all interfaces

(S104)